docs/GettingStarted/01-Why-Fedimint.md

```diff
@@ -36,9 +36,9 @@ We are building a solution which allows users to onboard to Bitcoin in a manner
 
 Fedimint allows bitcoiners to onboard new users, assisting them in their custody and payment model. Instead of referring a new bitcoiner to a third party custodian, you can onboard them yourself as part of a Federation.
 
-Put another way it allows you to **be your mum's / friends / villages bank**.
+Put in another way, it allows you to **be your mum's / friends / villages bank**.
 
-We call these close, trusted relationships "2nd party custodians". Fedimint federation guardians should be close friends and family members that you know personally, and can directly influence should they ever attempt to violate your trust.
+We call these close, trusted relationships "2nd party custodians".Fedimint federation guardians should be individuals you know personally, such as close friends or family members, whom you can directly influence if they ever attempt to breach your trust.
```

Comment:
`custodians".Fedimint`: missing space again. I wonder if I can add a lint for that